### PR TITLE
Added versioning for bundle js, chunks and theme

### LINKED ISSUE #31: Add versioning for js and css files
Change the way the bundle is created by adding a hash key to the js and css files.
Do this only for production

see [2432](https://github.com/geosolutions-it/MapStore2/issues/2432)

### PR DESCRIPTION
**Pull request by MV88**
Fix #31 

Introduced changes to how the bundle is created adding the last commit as the version.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -5,7 +5,10 @@ if grep -q XXXXXXXXXX 'web/ldap.properties'; then
   exit 1
 fi
 
+export GITREV=`git log -1 --format="%H"`
+export VERSION="SNAPSHOT-$GITREV"
```
Comment (mbarto) [inline build.sh]: I would make the version overridable from a script parameter, for releases

File: new.json [hunk 2]
```diff
@@ -50,7 +50,20 @@
         "source": "OpenTopoMap",
         "group": "background",
         "visibility": false
-      }
+      },
```
Comment (mbarto) [inline new.json]: This is unrelated, right?

Comment (MV88) [reply new.json]: yes, but i would avoid to do another pr only for that

File: build.sh (hunk 1 above)
Comment (MV88) [reply build.sh]: ok, i'll add this if:

if [ $# -eq 0 ]
  then
    mvn clean install -Dmapstore2.version=$VERSION
  else
    mvn clean install -Dmapstore2.version=$1
fi
